testing/py3-plac/APKBUILD

```diff
@@ -0,0 +1,35 @@
+# Contributor: Oleg Titov <oleg.titov@gmail.com>
+# Maintainer: Oleg Titov <oleg.titov@gmail.com>
+pkgname=py3-plac
+pkgver=1.0.0
+pkgrel=0
+pkgdesc="Parsing the command line the easy way"
+url="https://github.com/micheles/plac"
+arch="noarch"
+license="BSD"
+depends="python3"
+makedepends="python3-dev py3-setuptools"
+checkdepends="py3-pytest py3-tox"
+subpackages="$pkgname-doc"
+source="https://github.com/micheles/plac/archive/plac-$pkgver.tar.gz
+tox.ini"
+builddir="$srcdir/plac-plac-$pkgver"
+
+build() {
+	python3 setup.py build
+}
+
+check() {
+	# some tests will fail if sqlsoup is not installed and it is not installed
```

Comment:
Why not install it then?

Reply:
Upstream handled it this way, https://github.com/micheles/plac/blob/1.0.0/README.rst#testing :
`Some tests will fail if sqlsoup is not installed. Run a pip install sqlsoup or just ignore them.`

So I have not installed `sqlsoup` and ignored errors. `sqlsoup`  is not available as a package for Alpine yet.